pymc/distributions/continuous.py

```diff
@@ -1081,7 +1081,9 @@ class Beta(UnitContinuous):
     mu : tensor_like of float, optional
         Alternative mean (0 < ``mu`` < 1).
     sigma : tensor_like of float, optional
-        Alternative standard deviation (1 < ``sigma`` < sqrt(``mu`` * (1 - ``mu``))).
+        Alternative standard deviation (0 < ``sigma`` < sqrt(``mu`` * (1 - ``mu``))).
+    nu : tensor_like of float, optional
```

Comment:
Missing the relationship between the nu and alpha/beta in the math section above

Reply:
I added the relationship in the math section.